release_notes/372-release-notes.adoc

```diff
@@ -8,4 +8,265 @@ toc::[]
+
+| K8sRole
+| Label
+| Role Binding Label
+
+| K8sRoleAnnotation
+| Annotation
+| Role Binding Annotation
+
+|====
+
+[id="psp-changes"]
+=== PodSecurityPolicy (PSP) objects and Kubernetes 1.25
+
+The following changes were made in {product-title-short} to support the removal of PSP objects in Kubernetes 1.25:
+
+* PSP objects are created by default in installations for backward compatibility with Kubernetes versions prior to 1.25. However, you can manually disable PSP creation by setting `system.enablePodSecurityPolicies = false` in the Helm chart. 
+* Beginning with {product-title-short} version 3.71, auto-sensing has been added to {product-title-short} Helm charts. If you install {product-title-short} using the Operator or Helm charts, and {product-title-short} detects a Kubernetes version of 1.25 or later, it will not install PSPs. If you are using the `roxctl` CLI to install {product-title-short}, you need to disable PSP usage by setting the `--enable-pod-security-policies` flag to `false` for the `roxctl central generate` and `roxctl sensor generate` commands.
+
+Kubernetes users must disable the admission controller plugin for PSPs before upgrading to Kubernetes version 1.25. 
+
+
+[id="upcoming-api-changes"]
+=== Notice of upcoming API changes
+
+* CSV export: Beginning in the {product-title-short} 3.73 release, the CSV export `API /api/vm/export/csv` will require the `CVE Type` filter as part of the input query parameter. Requests that do not have the filter will give an error. Examples:
+** `CVE Type:NODE_CVE`
+** `CVE_Type:IMAGE_CVE`
+** `CVE_TYPE:K8S_CVE`
+* Suppress and unsuppress payloads: 
+** The field `ids` in the `/v1/cves/suppress` and `/v1/cves/unsuppress` API payloads will be renamed to `cves` in the {product-title-short} 3.73 release.
+** The `cves.ids` field of the `storage.VulnerabilityRequest` object in the response of `VulnerabilityRequestService` endpoints will be renamed to `cves.cves` in the {product-title-short} 3.73 release.
+
+[id="known-issues"]
+== Known issues
+
+{product-title-short} shows the wrong severity when two severities exist for a single vulnerability in a single distribution. This issue occurs because {product-title-short} scopes severities by namespace rather than component. There is no workaround. (ROX-12527)
+
+[id="bug-fixes-372"]
+== Bug fixes
+
+* Before this update, the steps to configure {ocp} OAuth for more than one URI were missing. The documentation has been revised to include instructions for configuring OAuth in {ocp} to use more than one URI. For more information, see xref:../operating/manage-user-access/configure-ocp-oauth.adoc#create-additional-routes-ocp-oauth_configure-ocp-oauth[Creating additional routes for the OpenShift Container Platform OAuth server]. (ROX-11296) 
+* Before this update, the autogenerated image integration (for example, a Docker registry integration) for a cluster is not deleted when the cluster is removed from Central. This issue is fixed. (ROX-9398)
```

Comment:
Does a user know what _Central_ is in this context?

Reply:
I think it should be OK as is - it's one of the two major pieces of ACS and is usually just referred to as "Central" (no qualifiers).